packages/trader/src/Modules/Trading/Containers/__tests__/toast-popup.spec.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../trader-providers';
+import { ToastPopup, NetworkStatusToastErrorPopup } from '../toast-popup';
+
+const mocked_props_toast_popup = {
+    portal_id: 'popup_root_1',
+    children: <div>test</div>,
+    className: '',
+    is_open: true,
+    onClose: jest.fn(),
+    onClick: jest.fn(),
+    type: 'info',
+    timeout: 0,
+};
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Toast: () => <div>Toast component</div>,
+    MobileWrapper: () => <div>MobileWrapper component</div>,
+}));
+
+describe('<ToastPopup />', () => {
+    let container;
+    beforeEach(() => {
+        document.body.innerHTML = '';
```

Comment:
I need modals with specific `id` (which I'll set in each test case), so it was better to create it manually then use  `jest.fn()`